### PR TITLE
completed two column layout-Bo Mercado

### PR DESCRIPTION
**Pull request by rbomercado**
image

### CODE REVIEW

**Comment by rbomercado** [comment]
oops this is in progress still sorry!

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Bo! Just submitting a review so that I know this is still in progress.

**Comment by rbomercado** [comment]
Gotcha! Fixed hopefully all that was off so to speak!
image

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+.width {
+  margin-left: 20%;
+}
+
+.half-columns {
+  display: flex;
+  margin-left: 12%;
+}
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:68]: Is this supposed to be a `column-half` class? If so, its styling should only be `width: 50%`.

File: css-two-column-layout/index.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        <div>
+          <img src="images/software.svg " alt="software">
+        </div>
+        <div>
+          <h3 class="col-4">INTUITIVE, SMART SOFTWARE</h3>
+          <p class="col-6">We build products that transform how healthcare facilities run across Africa
+      - making them radically more efficent operationally and financialy, a complete
+      turnaround.
+          </p>
+        </div>
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:32]: This should be one `column-half` element, since it takes up half the width of its row.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Bo, I suggest you review `css-layout-classes` again, you seem to still not fully comprehend their usage and how they should be made. If you have questions on this, please reach out, I am happy to help.

**Comment by rbomercado** [comment]
image


updated!

File: css-two-column-layout/index.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+<body>
+  <div class="container">
+   <div class=" margin-top  header-marg text-align-center">
+    <h1 class=" biggest-text col-10">SCALE,SPEED,AGILITY</h1>
+    <p class="col-9"> We've decided thousands of man hours to making healthcare institutions in the
+      Helium <br> network run 10x more efficiently and deliver an uparalled quality of care.
+    </p>
+    </div>
+   <div class="row">
+    <div class="col-5">
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:21]: `col-5` what? Is this meant to signify `column-fifth`? I would err on the side of being more descriptive with your class names.

File: css-two-column-layout/index.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+     <div class="row1">
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:22]: `row1`? Why do you need an additional class for `row` elements? If you want slightly different b behavior for a particular row, you should make a class you can add **_in addition_** to the normal `row` class. So you would end up with something like `class="row some-other-class"` on the element instead. That way you can use the original `row` class and just augment it with whatever special behavior you are trying to do.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-two-column-layout/styles.css [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+  width: 100%;
+  max-width: 1200px;
+}
+
+.row {
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  display: flex;
+  padding-left: 30px;
```
Comment (thebearingedge) [inline css-two-column-layout/styles.css:16]: Please remove this `padding-left`.

File: css-two-column-layout/styles.css [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+  padding: 10px 50px;
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-size: 18px;
+  font-weight: bold;
+}
+
+.col-1 {
+  max-width: 10%;
+}
```
Comment (thebearingedge) [inline css-two-column-layout/styles.css:76]: You can remove this if it's not being used.

File: css-two-column-layout/styles.css [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+
+.col-9 {
+  max-width: 90%;
+  margin-left: 8%;
+}
+
+.col-10 {
+  max-width: 100%;
+}
```
Comment (thebearingedge) [inline css-two-column-layout/styles.css:113]: Please remove any of these that aren't being used.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, great. Just about done. No rush on these minor changes.

**Comment by rbomercado** [comment]
updated selector properties and removed selectors that were not being used

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**